reactioncommerce/reaction
# Fix "Failed to get currency exchange rates" error in server

Resolves #3103 

This PR fixes the error logged on the server when the open exchange appId is not set. It ensures that the error message doesn't show up if appId isn't set.

## Test

1. Unset the Open Exchange API App ID by removing it from the `reaction.json` file
2. Do a `reaction reset` of the app
3. Run app after reset
4. Visit the home route `http://localhost:3000`
5. Check that on the terminal, you no longer see the error message "Failed to get currency exchange rates."
